src/transform-sdl.spec.ts

```diff
@@ -21,6 +21,26 @@ describe('transform-sdl', () => {
       }\n`);
   });
 
+  it('should add directives to interface sdl', () => {
+    expect(
+      addFederationAnnotations(
+        `
+      interface Product @keep {
+        id: Int
+      }`,
+        {
+          Product: {
+            keyFields: ['id'],
+            extend: true,
```

Comment:
Can we add a sanity check for externals too?

```suggestion
            extend: true,
            fields: {
              id: {
                external: true
              }
            },
```